lib/Parse/ParseType.cpp

```diff
@@ -545,38 +545,54 @@ bool Parser::parseGenericArguments(SmallVectorImpl<TypeRepr*> &Args,
 ///   type-identifier:
 ///     identifier generic-args? ('.' identifier generic-args?)*
 ///
-ParserResult<TypeRepr> Parser::parseTypeIdentifier() {
-  if (Tok.isNot(tok::identifier) && Tok.isNot(tok::kw_Self)) {
-    // is this the 'Any' type
-    if (Tok.is(tok::kw_Any)) {
-      return parseAnyType();
-    } else if (Tok.is(tok::code_complete)) {
-      if (CodeCompletion)
-        CodeCompletion->completeTypeSimpleBeginning();
-      // Eat the code completion token because we handled it.
-      consumeToken(tok::code_complete);
-      return makeParserCodeCompletionResult<IdentTypeRepr>();
-    }
+/// SWIFT_ENABLE_TENSORFLOW: Added `isParsingQualifiedDeclName` flag.
+/// The `isParsingQualifiedDeclName` flag controls whether parsing is done as if
```

Comment:
this comment block seems to better belong to the header / interface.

Reply:
Done.